src/db_client/standalone.rs

```diff
@@ -1,70 +1,43 @@
 // Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
 
+use std::sync::Arc;
+
 use async_trait::async_trait;
 
+use super::direct::DirectInnerClient;
 use crate::{
     db_client::DbClient,
     model::{
-        convert,
         request::QueryRequest,
         write::{WriteRequest, WriteResponse},
-        QueryResponse, Schema,
+        QueryResponse,
     },
-    rpc_client::{RpcClient, RpcContext},
-    Error, Result,
+    rpc_client::{RpcClientFactory, RpcContext},
+    Result,
 };
 
 /// Client for ceresdb of standalone mode.
 ///
 /// Now, [`StandaloneImpl`] just wraps [`RpcClient`] simply.
```

Comment:
The comment is outdated now, maybe change to
`Now, [`StandaloneImpl`] just wraps [`DirectInnerClient`] simply.`